testing/acme.sh/APKBUILD

```diff
@@ -2,23 +2,21 @@
 # Maintainer: Francesco Colista <fcolista@alpinelinux.org>
 pkgname=acme.sh
 pkgver=2.8.0
-pkgrel=0
+pkgrel=1
 pkgdesc="An ACME Shell script, an acme client alternative to certbot"
 url="https://github.com/Neilpang/acme.sh"
 arch="noarch"
 license="GPL-3.0"
-depends="curl"
+depends="curl openssl"
```

Comment:
`cmd:openssl` might suffice in this case.

Reply:
done